### PR TITLE
[Refactor] Replace ZMGenericMessage in ZMClientMessage tests

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

Replace ZMGenericMessage in:
- ZMClientMessage tests
- ZMMessage tests

### CODE REVIEW

**Comment by typfel** [comment]
This PR includes old changes. Is the target branch not merged into the PR branch?

File: Cartfile.resolved [hunk 1]
```diff
@@ -1,13 +1,13 @@
-github "wireapp/HTMLString" "4.1.0-beta.1-xcode_10_2"
+github "wireapp/HTMLString" "4.1.0-beta.1-xcode_11_4"
```
Comment (billypchan) [inline Cartfile.resolved:1]:
```suggestion
github "wireapp/HTMLString" "4.1.0-beta.1-xcode_11_4_1"
```

File: Cartfile.resolved [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-github "wireapp/swift-protobuf" "1.5.0"
-github "wireapp/wire-ios-cryptobox" "21.1.0"
-github "wireapp/wire-ios-images" "30.0.0"
-github "wireapp/wire-ios-link-preview" "26.0.0"
-github "wireapp/wire-ios-protos" "22.1.0"
-github "wireapp/wire-ios-system" "30.1.0"
-github "wireapp/wire-ios-testing" "21.0.0"
-github "wireapp/wire-ios-transport" "68.0.0"
-github "wireapp/wire-ios-utilities" "36.0.0"
+github "wireapp/swift-protobuf" "1.8.0"
```
Comment (billypchan) [inline Cartfile.resolved:5]:
```suggestion
github "wireapp/swift-protobuf" "1.8.0_Swift5.2.2"
```

**Comment by KaterinaWire** [comment]
> Looks like there are some Xcode 11 changes still visible in the diff?

@typfel I think you mean this one`SWIFT_OPTIMIZATION_LEVEL = "-O";`.
I reverted these changes.

**Pull request merged**